README.md

```diff
@@ -53,7 +53,7 @@ val pca = new com.nvidia.spark.ml.feature.PCA()
     $ make install                                      # install the library librmm.so to '/install/path'
     ```
 5. [RAFT](https://github.com/rapidsai/raft):
-    - raft provides only header files, so not build instructions for it.
```

Comment:
noticed only `PCA` was mentioned in this readme. Does it mean only this algorithm is supported or include all others from original apache package?

from line #24, does it mean all existing functions are supported? Is there any potential compatibility issue?
```
We used a customized class and add some extra API switches:
...
```

Reply:
Good catch. Now only PCA is supported now. I will update his confusing part.